base_location_nuts/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
-"access_res_partner_nuts_user","res_partner_nuts group_user","model_res_partner_nuts","base.group_user",1,0,0,0
+"access_res_partner_nuts_user_all","res_partner_nuts group_user_all","model_res_partner_nuts","",1,0,0,0
+"access_res_partner_nuts_user","res_partner_nuts group_user","model_res_partner_nuts","base.group_user",1,1,1,1
```

Comment:
This group is not correct. The one that has rights for creating states and countries is "Extra Rights / Contact Creation", so that one should be used here.

Reply:
`base.group_partner_manager`

Reply:
Done, sorry for the mistake... (i only saw permissions and not the group id)